app/controllers/donates_controller.rb

```diff
@@ -0,0 +1,29 @@
+class DonatesController < ApplicationController
+  helper_method :goal
+
+  def create
+    result = Goal::Donations::Create.call(donation_params)
```

Comment:
Regarding `Donations` - I thought we had a deal, that we use only singular form. Am I right ?